engine/crates/integration-tests/tests/federation/trusted_documents.rs

```diff
@@ -21,12 +21,6 @@ const TRUSTED_DOCUMENTS: &[TestTrustedDocument] = &[
         document_id: "favourite-repo-query-doc-id",
         document_text: "query { __typename }",
     },
-    TestTrustedDocument {
```

Comment:
The API of the trait changed in such a way that we can't test the wrong branch case anymore.